terraform-post-packer/main.tf

```diff
@@ -24,12 +24,12 @@ locals {
   account_name_regex  = format("^env[[:digit:]]+ \\(%s\\)$", local.images_account_type)
 }
 
-# The IDs of all cisagov/windows-commando-vm-packer AMIs
+# The IDs of all cisagov/windows-commando-packer AMIs
 data "aws_ami_ids" "historical_amis" {
   filter {
     name = "name"
     values = [
-      "example-hvm-*-x86_64-ebs",
+      "windows-server-2022-*-x86_64-ebs",
```

Comment:
Reminder that this name should change once we actually start installing Commando tools on here.